Git_GitHub/1_Git_and_GitHub.md

```diff
@@ -1,25 +1,28 @@
 # Git and GitHub
 
 Git is the most widely used version control software (VCS). It is a very powerful tool to know, and we are here to learn how to get started with it. 
-Git and GitHub are two different tools to facilitate version control, collaboration and sharing. It's important to understand that Git is not GitHub
+Git and GitHub are two different tools to facilitate version control, collaboration and sharing. It's important to understand that Git is not GitHub.
 
 Git is a software that operates locally at your machine. 
-A Git repository in Git, is a project in a "Folder". It contains all of the files and folders associated with your project and all the various versions of them over time. That helps to manage file changes. If you are just using Git for version control of the repository and you don't want to share your projects, you might never use GitHub.
+A repository in Git is a project in a "Folder". This folder contains all of the files and folders associated with your project and all the various versions of them over time. That helps to manage file changes. If you are just using Git for version control of the repository and you don't want to share your projects, you might never use GitHub.
 
 GitHub is a service that hosts Git repositories in the cloud. Users could share a repository, and collaborate at various steps of the project.
 
+**[DW]** all the three images below are not showing [/DW]
+**[MT]** I know it only shows up in the local machine. do you know how to fix it? [/MT]
```

Comment:
There are no figures in a fig subdirectory of this repository. The directory and the png need to be added.